ember-server/src/main/scala/org/http4s/ember/server/EmberServerBuilder.scala

```diff
@@ -213,9 +256,23 @@ object EmberServerBuilder {
     val port: Int = server.defaults.HttpPort
 
     def httpApp[F[_]: Applicative]: HttpApp[F] = HttpApp.notFound[F]
+
+    private val serverFailure =
+      Response(Status.InternalServerError).putHeaders(org.http4s.headers.`Content-Length`.zero)
+    // Effectful Handler - Perhaps a Logger
+    // Will only arrive at this code if your HttpApp fails or the request receiving fails for some reason
+    def errorHandler[F[_]: Applicative]: PartialFunction[Throwable, F[Response[F]]] = {
+      case (_: Throwable) =>
+        serverFailure.covary[F].pure[F]
+    }
+
+    @deprecated(
+      "0.21.17",
+      "Use errorOfLastResort for end of the line, or error handler for expected effectful handling")
```

Comment:
`errorOfLastResort` is gone now, right?  I think we just want to point people to `errorHandler` entirely?

Reply:
Yep. That's gone. Will fix.